CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -832,6 +832,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.3.0...v3.4.0) and
 - Add `etcd --experimental-peer-skip-client-san-verification` to [skip verification of peer client address](https://github.com/etcd-io/etcd/pull/10524).
 - Add `etcd --experimental-compaction-batch-limit` to [sets the maximum revisions deleted in each compaction batch](https://github.com/etcd-io/etcd/pull/11034).
 - Reduced default compaction batch size from 10k revisions to 1k revisions to improve p99 latency during compactions and reduced wait between compactions from 100ms to 10ms.
+- [print gRPC metadata in guarnteed order using the official go fmt pkg](https://github.com/etcd-io/etcd/pull/18311).
```

Comment:
Thanks for raisingthis @mohamedawnallah - As with the `CHANGELOG-3.5` pr, this change needs to be included under the section at the top for the next released version `v3.4.33` under a `### Package clientv3` header.

Reply:
Thanks! updated